shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -36,7 +50,18 @@ - (void) setupGo
 
 #if TESTING
 #else
-  NSString * logFile = [home stringByAppendingPathComponent:@"ios.log"];
+  NSString * library = [home stringByAppendingPathComponent:@"Library"];
+  NSString * appSupport = [library stringByAppendingPathComponent:@"Application Support"];
+  NSString * keybasePath = [appSupport stringByAppendingPathComponent:@"Keybase"];
+  NSString * logFile = [keybasePath stringByAppendingPathComponent:@"ios.log"];
```

Comment:
You can use slashes?

NSString * keybasePath = @"~/Library/Application Support/Keybase";